main/keys.go

```diff
@@ -7,6 +7,7 @@ const (
 	defaultString                   = "default"
 	configFileKey                   = "config-file"
 	versionKey                      = "version"
+	genesisConfigFileKey            = "genesis"
```

Comment:
I could be convinced to go with `genesisFileKey`

Reply:
I think this is fine as is